src/NzbDrone.Core/Download/Clients/rTorrent/RTorrentSettings.cs

```diff
@@ -52,10 +50,13 @@ public RTorrentSettings()
         [FieldDefinition(6, Label = "Category", Type = FieldType.Textbox, HelpText = "Adding a category specific to Sonarr avoids conflicts with unrelated downloads, but it's optional.")]
         public string TvCategory { get; set; }
 
-        [FieldDefinition(7, Label = "Recent Priority", Type = FieldType.Select, SelectOptions = typeof(RTorrentPriority), HelpText = "Priority to use when grabbing episodes that aired within the last 14 days")]
+        [FieldDefinition(7, Label = "Directory", Type = FieldType.Textbox, HelpText = "Optional location to put downloads in, leave blank to use the default rTorrent location")]
+        public string TvDirectory { get; set; }
```

Comment:
Advanced makes sense. I was thinking for validation we could check if its localhost/127.0.0.1 and validate then, otherwise don't check it. I like the idea of making sure its not a series folder/root folder (or drone factory?), though its possible they use the same path on different systems (seems weird, but you never know..), in that case just block it?